sample-tracking-status-overview-app/src/main/resources/life/qbic/portal/sampletracking/view/samples/SampleDesign.html

```diff
@@ -21,23 +21,6 @@
      </vaadin-button>
     </vaadin-horizontal-layout>
    </vaadin-horizontal-layout>
-   <vaadin-grid size-full data-item-type="life.qbic.business.samples.Sample" selection-mode="none" _id="sampleGrid" :middle :center :expand>
-    <table>
-     <colgroup>
-      <col column-id="name" sortable>
-      <col column-id="code" sortable>
-      <col column-id="status" sortable>
-     </colgroup>
-     <thead>
-      <tr default>
-       <th plain-text column-ids="name">Sample Label</th>
-       <th plain-text column-ids="code">QBiC Code</th>
-       <th plain-text column-ids="status">Status</th>
-      </tr>
-     </thead>
-     <tbody></tbody>
-    </table>
-   </vaadin-grid>
```

Comment:
what was this?

Reply:
That was the previous grid. It was created with the vaadin designer. Unfortunately, as we do not have a licence anymore, we cannot modify this. Thus I had to do it in java.